diff --git a/node_modules/expo-detox-hook/scripts/build_framework.ios.sh b/node_modules/expo-detox-hook/scripts/build_framework.ios.sh
index bbfc35d..37a7245 100755
--- a/node_modules/expo-detox-hook/scripts/build_framework.ios.sh
+++ b/node_modules/expo-detox-hook/scripts/build_framework.ios.sh
@@ -19,7 +19,7 @@ function prepareAndBuildFramework () {
     sourcePath="${rootPath}"/ios_src
     extractSources "${sourcePath}"
     buildFramework "${sourcePath}"
-    rm -fr "${sourcePath}"
+    # rm -fr "${sourcePath}"
   fi
 }
 
@@ -42,7 +42,7 @@ function main () {
     if [ ! -d "${frameworkPath}" ]; then
       echo "${frameworkDirPath} was found, but could not find ExpoDetoxHook.framework inside it. This means that the ExpoDetoxHook framework build process was interrupted.
          deleting ${frameworkDirPath} and trying to rebuild."
-      rm -rf "${frameworkDirPath}"
+      # rm -rf "${frameworkDirPath}"
       prepareAndBuildFramework
     else
       echo "ExpoDetoxHook.framework was previously compiled, skipping..."
